src/PKSim.Core/Services/ParameterUpdater.cs

```diff
@@ -101,17 +101,15 @@ private ICommand updateDistributedTableFormula(IParameter targetParameter, IDist
       private ICommand setDistributionValue(IParameter sourceParameter, IDistributedParameter targetParameter)
       {
          //distributed to distributed
-         var sourceDistributedParameter = sourceParameter as IDistributedParameter;
-         if (sourceDistributedParameter != null)
-            return setParameterValue(sourceParameter, targetParameter, false);
+         if (sourceParameter is IDistributedParameter sourceDistributedParameter)
+            return setParameterValue(sourceDistributedParameter, targetParameter, false);
 
-         //source parameter it not distributed. Is the formula a table paramter? In that case, we are most likely
+         //source parameter it not distributed. Is the formula a table parameter? In that case, we are most likely
          //updating from simulation table parameter to distributed parameter in individual
-         var tableFormula = sourceParameter.Formula as DistributedTableFormula;
-         if (tableFormula == null)
-            return setParameterValue(sourceParameter, targetParameter, false);
+         if (sourceParameter.Formula is DistributedTableFormula tableFormula)
+            return _parameterTask.SetParameterPercentile(targetParameter, tableFormula.Percentile);
```

Comment:
Don't understand this change.
Before: if parameter formula is DistributedTableFormula: **setParameterValue(...)**
Now: if parameter formula is DistributedTableFormula: **SetParameterPercentile(...)**
To be honest, I also forgotten what **DistributedTableFormula** is.

Reply:
It's the same code. Just inverted the if...

Reply:
ah yes, you are right :)